cherryontech/unmatched
# Feature/um 13 tag filter update

## Description


This PR adds more tag groups to the current tags. removes the 'Asian' tag , removes Subrahmanyan Chandrasekhar from list of professionals , and re-groups the professionals accordingly. The width occupied by the tags is also limited so it wraps around to the next line.


## Related Tickets & Documents



## What type of PR is this? (check all applicable)


|     | Type                       |
| --- | -------------------------- |
|     | :sparkles: New feature     |
|  ✓   | 🎨 Style                   |
|  ✓   | :hammer: Refactoring       |
|     | 🔥 Performance Improvements|
|     | :bug: Bug fix              |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |


## Mobile & Desktop Screenshots/Recordings

<img width="1509" alt="Screenshot 2023-11-11 at 5 59 09 PM" src="https://github.com/cherryontech/unmatched/assets/110904882/51e8b8b0-4d51-4011-b922-a28da4bfed3e">


## What I learned


## Testing steps

Switch to this branch with the following command: git switch feature/UM-13-tag-filter-update
Install dependencies with the following command: npm i
Run the following scripts: npm run dev to start Vite's dev build & npm run tailwind to start the Tailwind CSS build
View the app in browser (the npm run dev script will return the url you can see the app at)
Navigate to the the Resources page and see newly added tags and tag layout 


## Added to documentation?

|     | Type                       |
| --- | -------------------------- |
|     | 📜 README.md    |
| ✓    | 🙅 no documentation needed    |


## What gif best describes this PR or how it makes you feel?
